fortanix-sgx-tools/src/bin/ftxsgx-runner.rs

```diff
@@ -59,6 +64,13 @@ fn main() -> Result<(), Error> {
         None => (),
     }
 
+    enclave_builder.arg(file);
+    if let Some(enclave_args) = args.values_of("enclave-args") {
+        for a in enclave_args {
```

Comment:
Can't you use the `args` method here after mapping `enclave_args`?
`args.values_of("enclave-args").map(|args| enclave_builders.args(args))`

Reply:
I'm not a fan of writing things with side effect is a functional style, but I can definitely use the `args` method instead of the for loop.